sf2e.mjs

```diff
@@ -2,7 +2,10 @@ const MODULE_ID = "starfinder-field-test-for-pf2e";
 
 function applyTheme(themeId) {
   const headEl = document.head;
-  headEl.querySelector(`link[href*=${MODULE_ID}]`)?.remove();
+
```

Comment:
This isn't necessary

Reply:
I've reverted this to how it was previously.